### PR TITLE
Manual merge

### PR DESCRIPTION
**Pull request by greg0ire**
Don't merge, just approve yadda yadda.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,23 +17,23 @@
...
-        "symfony/form": "^2.8 || ^3.0",
-        "symfony/framework-bundle": "^2.8 || ^3.0",
-        "symfony/security": "^2.8 || ^3.0",
-        "symfony/security-acl": "^2.8 || ^3.0"
+        "symfony/console": "^2.8 || ^3.2",
+        "symfony/doctrine-bridge": "^2.8 || ^3.2",
+        "symfony/form": "^2.8 || ^3.2",
+        "symfony/framework-bundle": "^2.8 || ^3.2",
+        "symfony/security": "^2.8 || ^3.2",
+        "symfony/security-acl": "^2.8 || ^3.2"
```
Comment (jordisala1991) [inline composer.json]: security-acl must be max 3.0 (3.2 does not exists)

Comment (greg0ire) [reply composer.json]: Thanks for the tip

**Comment by OskarStark** [comment]
ping @greg0ire

**Comment by greg0ire** [comment]
@OskarStark I was sleeping :P

**Pull request merged**